fabtests/man/man1/fi_unmap_mem.1

```diff
@@ -0,0 +1 @@
+.so man7/fabtests.7
```

Comment:
The test also needs to be added to runfabtests.sh script.

Reply:
I wasn't sure about that, since it will only really test a MR cache.  Do the Intel and AWS CI's use that script?

Reply:
Intel CI uses the script to run all fabtests. I think it would be good to have it as part of the test suite.

Reply:
AWS CI also uses that script. https://github.com/awslabs/libfabric-ci-scripts/blob/master/single-node.sh